lnwallet/chanfunding/coin_select.go

```diff
@@ -156,10 +156,13 @@ func sanityCheckFee(totalOut, fee btcutil.Amount, maxFeeRatio float64) error {
 			"got %.2f", maxFeeRatio)
 	}
 
+	maxFee := btcutil.Amount(int64(float64(totalOut) * maxFeeRatio))
```

Comment:
nit: the cast to `int64` isn't needed as we cast to `btcutil.Amount` anyway (which is an alias for `int64`).